docs/conf.py

```diff
@@ -54,16 +53,18 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.napoleon',
     'sphinx.ext.autosummary',
+    'sphinx.ext.viewcode',
     # 'plot_generator',
     'matplotlib.sphinxext.plot_directive',
     'sphinx_autodoc_typehints',  # needs to be after napoleon
     # 'ipython_directive',
     # 'ipython_console_highlighting',
     # 'scanpydoc',  # scanpydoc.elegant_typehints causes full doc rebuilds
-    'scanpydoc.rtd_github_links',
-    'scanpydoc.theme',
     'scanpydoc.definition_list_typed_field',
     'scanpydoc.autosummary_generate_imported',
+    "sphinx_design",
+    "sphinxext.opengraph",
```

Comment:
What does this give us?

Reply:
sphinx design -- nothing yet, but would be nice to have some cards included from there

open graph -- this gives link preview when you share a doc link online (e.g., twitter)